visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts

```diff
@@ -478,19 +479,19 @@ describe("nodeContextMenuController", () => {
 	describe("onBodyLeftClickHideNodeContextMenu", () => {
 		it("should not hide if a click on color-picker occurs", () => {
 			const broadcastHideEventSpy = jest.spyOn(NodeContextMenuController, "broadcastHideEvent")
-			const mockedMouseEvent = {
+			const mockedMouseEvent: any = {
```

Comment:
why do we define and not cast to any?

Reply:
I tried to specify the concrete type but it was difficult to so, so I went back to any. Defining something is always better than casting it, since we might otherwise override things that we did not intend to do (with any there's no difference but with other types there might be).